sdk/cosmos/azure-cosmos/azure/cosmos/aio/_container.py

```diff
@@ -993,7 +1003,12 @@ async def delete_item(
         await self.client_connection.DeleteItem(document_link=document_link, options=request_options, **kwargs)
 
     @distributed_trace_async
-    async def get_throughput(self, **kwargs: Any) -> ThroughputProperties:
+    async def get_throughput(
+            self,
+            *,
+            response_hook: Optional[Callable[[CaseInsensitiveDict, List[Dict[str, Any]]], None]] = None,
```

Comment:
sorry one more: Took another pass -- it looks like different functions have different typing for response_hook, just want to confirm this is correct

Example is here the type includes a List of dictionaries instead of AsyncItemPaged like functions below